tests/e2e/env/setup.sh

```diff
@@ -2,6 +2,14 @@
 
 set -e
 
+# Check for script dependencies.
+# Exit if dependencies are not met.
+if ! command -v jq &> /dev/null
```

Comment:
On previous versions of the setup script, we didn't make use of the jq library. It was introduced while fixing the subscriptions download bug. Now we use it for downloading all the plugins.

On GH Actions, jq comes pre-installed. On Macs, it isn't. So it will throw an error if jq isn't installed locally. Hence I added the dependency check